core-services/prow/02_config/openshift/console-plugin-template/_prowconfig.yaml

```diff
@@ -0,0 +1,121 @@
+tide:
+  queries:
+  - includedBranches:
```

Comment:
this whole block can be dropped, since we only got `main` branch, so there are no branches to backport to.

Reply:
So I think we only need following query:
```
 - includedBranches:
    - main
    - master
    labels:
    - approved
    - docs-approved
    - lgtm
    - px-approved
    - qe-approved
    missingLabels:
    - backports/unvalidated-commits
    - bugzilla/invalid-bug
    - do-not-merge/hold
    - do-not-merge/invalid-owners-file
    - do-not-merge/work-in-progress
    - jira/invalid-bug
    - keep-main-query-separate
    - needs-rebase
    repos:
    - openshift/console-plugin-template
```

everything else could be dropped